sideboard/lib/_cp.py

```diff
@@ -10,22 +10,61 @@
 import cherrypy
 
 import sideboard.lib
-from sideboard.lib import log
+from sideboard.lib import log, config
 
 _startup_registry = defaultdict(list)
 _shutdown_registry = defaultdict(list)
 
 
-def on_startup(func, priority=50):
+def _on_startup(func, priority):
```

Comment:
As explained in the docstring below: previously you could use ``@on_startup`` (or ``@on_shutdown``) in one of two ways:

1) As a decorator, e.g. ``@on_startup``.

2) Calling the function manually:

```python
def foo(): ...
on_startup(foo, priority=75)
```

Previously the second way was the only way you could specify a custom ``priorty`` for the function.

Now we have a third way:

```
@on_startup(priority=75)
def foo(): ...
```

In other words, you can now specify the priority when invoking this as a decorator.  This would make a couple of our MAGFest plugins slightly more elegant if we start making use of it.